nistats/thresholding.py

```diff
@@ -88,20 +94,26 @@ def map_threshold(stat_img=None, mask_img=None, alpha=.001, threshold=3.,
     # Check that height_control is correctly specified
     if height_control not in ['fpr', 'fdr', 'bonferroni', None]:
         raise ValueError(
-            "height control should be one of ['fpr', 'fdr', 'bonferroni', None]")
+            "height control should be one of "
+            "['fpr', 'fdr', 'bonferroni', None]")
 
+    # if two-sided, correct alpha by a factor of 2
+    alpha_ = alpha
+    if two_sided:
+        alpha_ = alpha / 2
+    
     # if height_control is 'fpr' or None, we don't need to look at the data
     # to compute the threhsold
     if height_control == 'fpr':
-        threshold = norm.isf(alpha)
+        threshold = norm.isf(alpha_)
```

Comment:
~`alpha_`~  `alpha_corrected`

Reply:
So on in the rest of the code.

Reply:
No, we should not name it 'alpha_corrected', which alludes to something else in this context.
To me the trailing _ simply means that the value had to be adapted etc. Why is it bad ?

Reply:
It would be clearer to have a more explicit and self-documenting name. If not `alpha_corrected` then perhaps `alpha_adapted`?